doc/man/lttng-create.1.txt

```diff
@@ -9,25 +9,24 @@ lttng-create - Create an LTTng tracing session
 
 SYNOPSIS
 --------
-Normal mode:
+Local or relay mode:
```

Comment:
> We might actually want to present it as a separate "mode"? WDYT?

I had this in mind, but since `--set-url` exists in both modes, I thought it was weird. But the synopsis of the local mode could be changed to this:

```
lttng [GENERAL OPTIONS] create [--shm-path=PATH]
      [--no-output | --output=PATH | --set-url=file://PATH] [SESSION]
```

WDYT?